src/acquisition/covid_hosp/update.py

```diff
@@ -0,0 +1,176 @@
+"""
+Acquires the "COVID-19 Reported Patient Impact and Hospital Capacity by State
+Timeseries" dataset provided by the US Department of Health & Human Services
+via healthdata.gov.
+"""
+
+# standard library
+import json
+import re
+
+# first party
+from delphi.epidata.acquisition.covid_hosp.database import Database
+from delphi.epidata.acquisition.covid_hosp.network import Network
+
+
+class UpdateException(Exception):
+  """Exception raised exclusively by the Update class."""
+
+
+class Update:
+
+  # regex to extract issue date from revision field
+  # example revision: "Mon, 11/16/2020 - 00:55"
+  REVISION_PATTERN = re.compile(r'^.*\s(\d+)/(\d+)/(\d+)\s.*$')
+
+  def get_entry(obj, *path):
```

Comment:
I'm not sure what you're referring to here.

(But yes, overkill is most likely an accurate description in any case :) )

Reply:
I meant writing a utility to dig through a deep tree, instead of just accessing it directly. It _does_ give us more sensical error messages, should they ever occur, and helps centralize the error handling implementation, but `get_entry` is only ever called six times, and that's including the tests. So... borderline, yeah. We'll leave it in.

Reply:
Yeah so I normally wouldn't go _quite_ to this extent, but in a recent PR in another repo I had some feedback asking for more graceful handling when things aren't found — so I went extra safe-mode here as well.